lhotse/features/io.py

```diff
@@ -951,26 +951,28 @@ def write(self, key: str, value: np.ndarray) -> str:
 @register_reader
 class KaldiReader(FeaturesReader):
```

Comment:
To avoid breaking compatibility for users who used these readers/writers before, it'd be better to copy and modify those classes (`KaldiNativeReader`/`Writer`) instead, and make them default everywhere instead. It also makes sense to add a deprecation warning to `KaldiReader` and `KaldiWriter` pointing to the new implementation.

Reply:
How about keeping the name fixed, i.e., still using the name kaldiio? As we only change the implementation detail, it should be backward compatible.

Reply:
makes sense!

Reply:
Reverted to `kaldiio` for reader/writer `type`.